README.md

```diff
@@ -38,7 +38,7 @@ TODO: Add volume mount for SQLite store
 
 ```text
 docker run -d \
-    -p 80:80 \
+    -p <your_desired_host_port>:8080 \
```

Comment:
WT Connector starts on port 8080 by default

Reply:
I think you can simply go 8080:8080. People can lookup docker run docs to figure out which is which.